rebasehelper/checkers/abipkgdiff_tool.py

```diff
@@ -40,6 +40,7 @@ class AbiCheckerTool(BaseChecker):
     DEFAULT = True
     results_dir = ''
     log_name = 'abipkgdiff.log'
+    abi_changes = False
```

Comment:
Just a heads up, this won't work. This attribute is only set to `False` when the module is imported for the first time, so if you execute `checkers_runner.run_checker()` multiple times and there are abi changes detected in one of the runs, they will be reported also for all consecutive runs.

Reply:
Well is not it enough, that ABI change appears in either one of all the (sub)packages, to warn the user to pay attention to it?

Reply:
Sure, but I'm talking about something different. It's wrong to assume there is only one rebase per process. Just try to execute `Application.run()` more than once in one script (with different packages of course) and see for yourself. With your approach we could end up having reports for packages containing information from totally different and unrelated packages (yes, it already happened).

Reply:
Oh sure I remember, then you're right, that needs fix as well.